setup.py

```diff
@@ -105,6 +105,14 @@
     "psycopg[binary,pool]>=3.0.0,<4",
 ]
 
+PROMETHEUS_CLIENT_REQUIRED = [
```

Comment:
I'm also a little unsure about these two, feels too granular. What about changing to a single extra named `telemetry` that will install both `prometheus_client` and `psutil`.

Reply:
Can you call it just `opentelemetry` instead of `opentelemetry_required`?